beps/0003-auth-architecture-evolution/README.md

```diff
@@ -378,6 +393,17 @@ router.get(
 );
 ```
 
+### Access Control Configuration
+
+In order to disable access control for a Backstage instance, the following configuration can be used:
+
+```yaml
+backend:
+  dangerouslyDisableServiceAuth: true
+```
+
+The exact impact that this has is that disabled the check in the `HttpRouterService` implementation, effectively applying the `unauthenticated` access level to all routes. Furthermore, it will also change `AuthService` so that the `issueServiceToken()` method will now issue an empty token for a `'none'` principal, rather than throwing.
```

Comment:
```suggestion
The exact impact that this has is that it disables the check in the `HttpRouterService` implementation, effectively applying the `unauthenticated` access level to all routes. Furthermore, it will also change `AuthService` so that the `issueServiceToken()` method will now issue an empty token for a `'none'` principal, rather than throwing.
```